src/audits/performance/input-readiness-metric.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('../audit');
+
+/* global window */
+window.global = window;
+
+// we need gl-matrix and jszip for traceviewer
+// since it has internal forks for isNode and they get mixed up during
+// browserify, we require them locally here and global-ize them.
+// from catapult/tracing/tracing/base/math.html
+var glMatrixModule = require('gl-matrix');
```

Comment:
> Can we shove all this into some other place and keep the funkiness out of the audit as much as possible?

sure.  i moved it into tracing processor.

https://github.com/GoogleChrome/lighthouse/compare/paullewis-converter...paullewis-converter-refactorintotracingprocessor

want me to put this into this PR or do it as a followup?